docs/getting-started.md

```diff
@@ -158,7 +162,7 @@ class Smarty_GuestBook extends Smarty {
 }
 ```
 
-Now, we can use `Smarty_GuestBook` instead of `Smarty` in our scripts:
+Now, we can use `My_GuestBook` instead of `Smarty` in our scripts:
 ```php
 <?php
 $smarty = new Smarty_GuestBook();
```

Comment:
should be :
$smarty = new My_GuestBook();

Reply:
thanx, fixed it.